src/monster.h

```diff
@@ -525,6 +524,9 @@ class monster : public Creature
         std::bitset<NUM_MEFF> effect_cache;
         cata::optional<time_duration> summon_time_limit = cata::nullopt;
 
+        player *find_dragged_foe();
+        void operate( player *dragged_foe ); // Nursebot operations
```

Comment:
Please don't repeat comments all over the place (here, at the definition of the function and at the invocation of it).

In fact, just name it `nursebot_operate` or similar and you don't even need a comment at all.

Reply:
Function renamed.